node/src/service/evm.rs

```diff
@@ -540,46 +534,13 @@ where
 			transaction_pool,
 			sync_service.clone(),
 			params.keystore_container.keystore(),
-			force_authoring,
-		)?;
-
-		let spawner = task_manager.spawn_handle();
-
-		let params = StartCollatorParams {
-			para_id: id,
-			block_status: client.clone(),
-			announce_block,
-			client: client.clone(),
-			task_manager: &mut task_manager,
-			relay_chain_interface,
-			spawner,
-			parachain_consensus,
-			import_queue: import_queue_service,
-			collator_key: collator_key.ok_or_else(|| {
-				sc_service::error::Error::Other("Collator Key is None".to_string())
-			})?,
 			relay_chain_slot_duration,
-			recovery_handle,
-			sync_service,
-		};
-
-		#[allow(deprecated)] // TODO fix before v1.3.0
-		start_collator(params).await?;
-	} else {
-		let params = StartFullNodeParams {
-			client: client.clone(),
+			para_id,
+			collator_key.expect("Command line arguments do not allow this. qed"),
+			overseer_handle,
 			announce_block,
-			task_manager: &mut task_manager,
-			para_id: id,
-			relay_chain_interface,
-			relay_chain_slot_duration,
-			import_queue: import_queue_service,
-			recovery_handle,
-			sync_service,
-		};
-
-		#[allow(deprecated)] // TODO fix before v1.3.0
-		start_full_node(params)?;
+		)?;
+	} else {
```

Comment:
Great catch! Can be removed because it represents a fullnode which doesn't need to to consensus work. Base: https://github.com/paritytech/polkadot-sdk/blob/0d7ca59fd4b4d15f9400971f32bc6a51b2fe837f/cumulus/parachain-template/node/src/service.rs#L304